pkg/output/output.go

```diff
@@ -170,11 +170,17 @@ func BadgeColor(value float64, levels *Levels) badge.Color {
 
 	return color
 }
-func NoRetestBadgeColor(value float64, levels *Levels) badge.Color {
+func NoRetestBadgeColor(value float64, mergedPRs float64, levels *Levels) badge.Color {
 	var color badge.Color
-	if value > levels.Yellow {
+	var percentVal float64
+	if value == 0 {
+		percentVal = 0
+	} else {
+		percentVal = value / mergedPRs
```

Comment:
Should we check for `mergedPRs == 0` here to avoid NaN cases?

Reply:
Yes good catch - should be rare but could happen. I will add it.

Reply:
Added - https://github.com/kubevirt/ci-health/compare/33acc30b61b413dff806b974d4a5aff523564cf1..ac7330d1774dca725b1027ba0d2f111e0bf02e57